docs/how_to/enabling_cross_sut_access.md

```diff
@@ -0,0 +1,23 @@
+# Enabling access bewteen SUTs during an acceptance test run
+
+If you are running acceptance tests for Beaker that, at some point, will perform one of the following:
+
+* SSH between SUTs
+* Clone private repos
+
+You will need to run an SSH agent, and add the SSH key for accessing your SUTs/private repos, prior to running the tests.
+
+To load the SSH agent and add your SSH key, run the following:
+
+~~~bash
+eval `ssh-agent`
+ssh-add <SSH key file path>
+~~~
+  
+Now run your acceptance tests:
+
+~~~bash
+bundle exec rake test:subcommands
```

Comment:
ugh, appears that the link only works if you already have the old change open. It's @nwolfe's first comment